test/Microsoft.ML.Tests/OnnxConversionTest.cs

```diff
@@ -261,8 +261,7 @@ public void TestVectorWhiteningOnnxConversionTest()
             Done();
         }
 
-        [Fact]
-        public void PlattCalibratorOnnxConversionTest()
+        private (MLContext, IDataView, List<IEstimator<ITransformer>>, EstimatorChain<NormalizingTransformer>) GetEstimatorsForOnnxConversionTests()
```

Comment:
Why again are testing Platt with all these estimators? Do they affect the way the model is converted?

Reply:
I believe we're testing not just Platt but now the FixedPlatt and and Naive estimators to ensure they're exported correctly and produce the same result as in C#.